dvc/output/base.py

```diff
@@ -228,11 +230,11 @@ def save(self):
             return
 
         if self.is_in_repo:
-            if self.repo.scm.is_tracked(self.path):
+            if self.repo.scm.is_tracked(self.fspath):
```

Comment:
not related to this PR. should all the stuff that depends on local-specific things go into outputLocal?

Reply:
Probably, but it's hard to decouple that from base code.

Reply:
This will get lost here) Should that go to a separate issue?

Reply:
@Suor Sure, please create an issue for refactoring :)